src/components/core/accordion/accordion-content.tsx

```diff
@@ -34,7 +34,7 @@ const Content = styled('div', {
     },
 });
 
-const AccordionContent = ({ children, expand_section, size }: AccordionContentProps) => {
+const AccordionContent = ({ children, expand_section, size, ...props }: AccordionContentProps) => {
```

Comment:
are the props used anywhere?

Reply:
@balakrishna-binary for instance if you want to use className for the content and add your own style , you need this. otherwise you must wrap it inside another div which I think is not a good.

Reply:
@iman-fs just a query, is there a need to style the content of accordion by providing className. The children that will be placed inside accordion, for them the style should be handled in children itself right? and also you can't see props being used

Reply:
Got it. So I'm closing the PR.